nominal/core/client.py

```diff
@@ -442,6 +443,50 @@ def get_commensurable_units(self, unit_symbol: str) -> Sequence[scout_units_api.
         """Get the list of units that are commensurable (convertible to/from) the given unit symbol"""
         return self._clients.units.get_commensurable_units(self._clients.auth_header, unit_symbol)
 
+    def get_logical_series(self, rid: str) -> timeseries_logicalseries_api.LogicalSeries:
+        """Get metadata for a given logical series by looking up its rid
+        Args:
+            rid: Identifier for the logical series to look up
+        Returns:
+            Resolved metadata for the requested logical series
+        Raises:
+            conjure_python_client.ConjureHTTPError: An error occurred while looking up the logical series.
+                This typically occurs when there is no such logical series for the given RID.
+        """
+        return self._clients.logical_series.get_logical_series(self._clients.auth_header, rid)
+
+    def set_series_units(
```

Comment:
The APIs use both "series" and "channel" for the same thing but we should use "channel" everywhere in the client (this is the decided/externally-facing name)